control-plane/agents/src/bin/core/controller/resources/operations_helper.rs

```diff
@@ -657,7 +657,9 @@ pub(crate) trait SpecOperationsHelper:
                 id: self.uuid_str(),
             })
         } else {
-            Err(SvcError::Deleting {})
+            Err(SvcError::Deleting {
+                kind: ResourceKind::Unknown,
```

Comment:
hmm if we can't figure out what resource we have then it's probably not worth having a kind here..

Reply:
It was set to `unknown` already in the translation to reply error.

Reply:
Indeed. That sounds good let me check that.

Reply:
We had a self.kind() already. Used that.

Reply:
Oh nice :D